### DISCUSSION TITLE
I can't get over: Connected. Hit any key..

### DISCUSSION THREAD

**Discussion by QuiXo** [jalbarracinv/python-cbm-petscii-bbs#2] [Q&A]
Hi,
After I prepare the database and start the server, I connect with SyncTERM but even if I press any key it does not go forward and disconnects due to timeout. Why do you think?

Keyboard input doesn't work. I tried to start the other functions like do_welcome or do_login and they start but I can't interact with the keyboard.

**Comment by jalbarracinv** [comment]
Hi, sorry for late answer. 
Do you see the message "Connected. Hit any key..."?
have you uploaded the /seq/welcome.seq file to the server?
After "pressing any key" you should see the welcome.seq contents
You can test it with my server at: wordle64.mywire.org   port:8864
